src/components/playlist/playlist.css

```diff
@@ -0,0 +1,92 @@
+/* 
+* simple approach with filter and different divs-level
+* backdrop-filter doesnt work with Firefox
+*/
+.playlist-container,
+.playlist-container-bg {
+  position: relative;
+  /* for testing purpose - 100% for real */
+  width: 188px;
+  height: 128px;
+}
+
+.playlist-background {
+  background-color: #aaa;
+  background-image: url(../../assets/playlist-img/marcela-laskoski-YrtFlrLo2DQ-unsplash.png);
+  width: 100%;
+  height: 100%;
+  filter: brightness(50%);
+}
+
+.playlist-info {
+  position: absolute;
+  top: 0px;
+  left: 0px;
```

Comment:
```suggestion
  top: 0;
  left: 0;
```

Reply:
i personally like it more with a unit, sometimes css is bitchy, but its shorter